### DISCUSSION TITLE
Other currencies for cryptocurrencies (such as BTC)

### DISCUSSION THREAD

**Discussion by sveetya** [ghostfolio/ghostfolio#3344] [Ideas]
I would love to see support for other currencies for crypto.

My current issue with using Ghostfolio:

- Currently, you can only add Bitcoin purchases using USD (using the BTC/USD ticker)
- So if you bought in other currencies such as EUR or CZK, you need to calculate the value equivalent into the USD using current exchange rate
- This approach causes inaccurate data since the current market value of BTC holdings is calculated into USD and not the original currency

**Comment by dtslvr** [comment]
Hi @Marilok

Thanks for sharing your idea. We already have a related ticket: https://github.com/ghostfolio/ghostfolio/issues/3320

I will close this discussion as a duplicate for better organization. Thanks for understanding!
